ajax/libs/collect.js/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "collect.js",
+  "filename": "collect.min.js",
+  "version": "4.0.11",
+  "description": "Convenient and dependency free wrapper for working with arrays and objects.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ecrmnn/collect.js"
+  },
+  "keywords": [
+    "collections",
+    "collection",
+    "collect",
+    "map",
+    "filter",
+    "group by",
+    "reduce",
+    "transform",
+    "pluck",
+    "pie",
+    "mapWithKeys",
+    "laravel",
+    "whereIn",
+    "values",
+    "unique",
+    "nested-arrays",
+    "laravel-collections",
+    "comparison",
+    "collectable"
+  ],
+  "author": {
+    "name": "Daniel Eckermann",
+    "email": "danieleckermann@gmail.com",
+    "url": "http://danieleckermann.com"
```

Comment:
Let's use `https`
Thanks

Reply:
It is not support `https`.

Reply:
Would you try again?
https://www.danieleckermann.com/
![2018-02-20 5 34 58](https://user-images.githubusercontent.com/8349587/36416614-7613dbae-1664-11e8-9275-def6f8777584.png)

Reply:
Thanks.